src/main/java/com/google/javascript/clutz/AliasMapBuilder.java

```diff
@@ -28,17 +28,18 @@
       return aliasMap;
     }
 
+    // Loop over the statements, looking for import statements, and build a map from the local variable
+    // names to the original symbol name

```

Comment:
add example:
' const C = goog.require('a.b.c');`
the map will contain: 'C' -> 'a.b.c'

Reply:
Done.